uitools/cpp/Esri/ArcGISRuntime/Toolkit/Internal/GeoViews.h

```diff
@@ -52,4 +46,30 @@ namespace Toolkit
 
 #endif
 
+namespace Esri {
+namespace ArcGISRuntime {
+namespace Toolkit {
+  inline Map* getGeoModel(MapViewToolkit* mapView)
+  {
+    return mapView ? mapView->map() : nullptr;
+  }
+
+  inline Scene* getGeoModel(SceneViewToolkit* sceneView)
+  {
+    return sceneView ? sceneView->arcGISScene() : nullptr;
+  }
+
+  inline auto getGeoModelChangedSignal(MapViewToolkit* /*mapView*/)
```

Comment:
💭  why do we need an arg here? seems like this could be templated rather than passing an arg we never use...

Reply:
This is for a template trick that allows for one to agnostically get the right signal based on some arbitrary object:

https://github.com/Esri/arcgis-runtime-toolkit-qt/blob/b6c0d18982a4faef43f7b368cbc5a329f93924dc/uitools/cpp/Esri/ArcGISRuntime/Toolkit/FloorFilterController.cpp#L120